tests/e2e/vm_connectivity_test.go

```diff
@@ -0,0 +1,205 @@
+package e2e
+
+import (
+	"fmt"
+	"github.com/deckhouse/virtualization/tests/e2e/executor"
+	"io/fs"
+	"log"
+	"os"
+	"path/filepath"
+	"sigs.k8s.io/yaml"
+	"strings"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	kc "github.com/deckhouse/virtualization/tests/e2e/kubectl"
+	virt "github.com/deckhouse/virtualization/tests/e2e/virtctl"
+)
+
+type Service struct {
+	ApiVersion string `yaml:"apiVersion"`
+	Kind       string `yaml:"kind"`
+	Metadata   struct {
+		Name      string `yaml:"name"`
+		Namespace string `yaml:"namespace"`
+	} `yaml:"metadata"`
+	Spec struct {
+		Ports []struct {
+			Port       int `yaml:"port"`
+			TargetPort int `yaml:"targetPort,omitempty"`
+			NodePort   int `yaml:"nodePort,omitempty"`
+		} `yaml:"ports"`
+	} `yaml:"spec"`
+}
+
+func getSVC(manifestPath string) *Service {
+	data, err := os.ReadFile(manifestPath)
+	if err != nil {
+		log.Fatalf("Error read file: %v", err)
+	}
+
+	var service Service
+	err = yaml.Unmarshal(data, &service)
+	if err != nil {
+		log.Fatalf("Error parsing yaml: %v", err)
+	}
+
+	return &service
+}
```

Comment:
this works if you have only one service. see GetVMFromManifest

Reply:
In this case, by design, I have only one service in yaml